CONTRIBUTING.md

```diff
@@ -0,0 +1,276 @@
+"what is build metadata?", or articulating what constitutes a prerelease, then you will probably be baffled by the
+restrictions on the workflow.
+
+1. `git flow release start $(invoke new-release --kind "major")`
+    - `invoke new-release` updates the VERSION file and echoes back the new version. There are three mutually exclusive
+      options for `invoke new release`, each appropriate for a different situation during the release cycle.
+        * `--kind`: `major`, `minor`, or `patch`. When creating a release branch, you'll be using this most often.
+        * `--pre`: Rarely, after starting a release branch, you want to deploy it to the staging environment for QA.
+          in this situation, you should **absolutely not call it in the context of the `git flow` tools**, since they
+          automate a workflow of merging _all_ release branches back into master and develop at the same time. I cannot
+          emphasize this enough - ***Do NOT, ever, under any circumstance, merge a prerelease branch into master!***
+          Instead, do the remote merge with develop manually:
+            - `invoke new-release --pre alpha`: Pre-releases proceed from *alpha*, to *beta*, to *rc* ("Release
+            Candidate"). You can iterate on a prerelease by just calling the command again; it'll go from something like
+            `1.2.1-beta.1` to `1.2.1-beta.2`. You can't go backwards - if you try to create a new alpha release after a
+            beta, the tool will quit and print an error message.
+            - `NEXT_LOOKIT_VERSION=$(<VERSION)`
+            - `git checkout develop`
+            - `git merge --no-ff release/$NEXT_LOOKIT_VERSION`
+            - `git tag -a $(cat VERSION) && git push origin --tags`: Tag the commit as a prerelease.
+            - `git push origin develop`: Push to remote and trigger the build.
+        * `--build`: Sometimes, we'll need to fix or change something with the [CI/CD pipeline][lookit-orchestrator], in
+          which case the likelihood is that you'll need to redeploy Lookit to re-initialize any environment variables
+          passed in from configs and secrets. While it's true that you could run `kubectl rollout restart
+          ${SOME_DEPLOYMENT}` to accomplish this, it's preferable here to annotate the commit history with valuable
+          information about the evolution of the product. Ideally, you'll be able to enter the commit SHA for the
+          updated version of [Lookit Orchestrator][lookit-orchestrator] that will be controlling your re-deployment.
+2. Now you're in a new release branch with the changed `VERSION` file in your git staging area (uncommitted). You should
+   make any changes you need to. Ideally, this is only additions to `CHANGELOG.md`.
+3. `git flow release finish $(cat VERSION)` - this will execute a `--no-ff` merge on both develop and master branches,
+   along with creating the appropriate tags both locally and remotely.
```

Comment:
is it possible to denote more clearly that this is essentially internal documentation - i.e. a public record of the process but not something you need to know or care about to contribute a feature? it seems intimidating to many people who'd be perfectly capable of handling a feature or bugfix, esp. bits like "you will probably be baffled by" versus explaining under what conditions you'd need to deal with this

Reply:
See above note about changes to headings